samples/java-customer-registry/pom.xml

```diff
@@ -17,8 +17,7 @@
     <akkasls.dockerTag>${project.version}</akkasls.dockerTag>
     <akkasls.mainClass>customer.Main</akkasls.mainClass>
     <jdk.target>11</jdk.target>
-    <akkaserverless-sdk.version>0.7.0-beta.10</akkaserverless-sdk.version>
-    <akkaserverless-maven-plugin.version>0.7.0-beta.10</akkaserverless-maven-plugin.version>
```

Comment:
sdk.version is enough. All artifacts must align to it.

Reply:
I had that on my mental note list but forgot to talk to anyone about it :)